### PR TITLE
Critique of Authority via @dextreFreeman

### PR DESCRIPTION
**Pull request by dirkkelly**
Merges in the great work from @dextreFreeman

critique-of-authority

### CODE REVIEW

File: data/videos.yml [hunk 1]
```diff
@@ -61,3 +61,52 @@ KlMlZTHShFw:
...
+  title: "DO #BLUELIVESMATTER ?"
+  channel: katblaque
+  description: In this video, I talk about Blue Lives Matter, Police Brutality and violence against the police. I also share my personal perspective on Law Enforcement.
+  series: katblaque
+  source: youtube
+LB3HUXdmid4:
+  id: LB3HUXdmid4
+  title: "Let's talk about police militarization...."
+  channel: beauofthefifthcolumn
+  description:
```
Comment (craiglonsdale) [inline data/videos.yml]: Probably want to throw in a description here for consistencies sake.

File: data/videos.yml [hunk 2]
```diff
@@ -61,3 +61,52 @@ KlMlZTHShFw:
...
+  title: "Is Thought Slime WRONG about Is Batman a Fascist? - OPPORTUNISTIC TAKEDOWN VIDEO!"
+  channel: noncompete
+  description: In this SPICY TAKEDOWN of the reprehensible THOUGHT SLIME I discuss the difference (and overlap) between gatekeeping and criticism.
+  series: NonCompete
+  source: youtube
+BmjB7TUroyE:
+  id: BmjB7TUroyE
+  title: "Let's talk about a cop asking me for training...."
+  channel: beauofthefifthcolumn
+  description:
```
Comment (craiglonsdale) [inline data/videos.yml]: Probably want to throw in a description here for consistencies sake.

File: data/videos.yml (hunk 1 above)
Comment (dirkkelly) [reply data/videos.yml]: @craiglonsdale agreed we should be striving for that, but it's hard when the source video doesn't have a description, we don't have a process for handling this yet. I think we can merge and add later. There is already an EarthStrike video without a description.

Comment (craiglonsdale) [reply data/videos.yml]: Ok fair enough. Maybe we can do something with the styling to handle no description in the future.

**Pull request merged**